composer.json

```
@@ -17,6 +17,7 @@
     },
     "require": {
         "php": "^7.1.3",
+        "symfony/config": "^4.2",
         "symfony/property-access": "^3.4 || ^4.0",
```

```suggestion
        "symfony/property-access": "^4.0",
```

let's also remove support for symfony 3.4 here, we'll bump a new major release anyway 

to make the CI build pass, we will have to tell `travis` to also stop building using `SYMFONY_VERSION: 3.4`... 
I'd suggest to use `SYMFONY_VERSION: 4.4` to be prepared for symfony 5